src/Command/Branch/BranchChangelogCommand.php

```diff
@@ -25,16 +27,31 @@ protected function configure()
     {
         $this
             ->setName('branch:changelog')
-            ->setDescription('Reports what got fixed or closed since last release on current branch')
+            ->setDescription('Reports what got fixed or closed since last release on the branch')
+            ->addArgument(
+                'branch',
+                InputArgument::OPTIONAL,
+                'Branch to look for tags, when none is given the current branch is used'
+            )
+            ->addOption(
+                'search',
+                's',
+                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+                'Regex pattern to use for searching', ['/#(?P<id>[0-9]+)/i']
+            )
             ->setHelp(
                 <<<EOF
-Reports what got fixed or closed since last release on current branch.
-reference: http://www.lornajane.net/posts/2014/github-powered-changelog-scripts
-
-The <info>%command.name%</info> command :
+Reports what got fixed or closed since the last release on current branch.
```

Comment:
Reports what has been fixed or closed since the last release on the given branch